tests/foreman/ui/test_registration.py

```diff
@@ -121,3 +128,496 @@ def test_negative_global_registration_without_ak(
+        lifecycle_environment=module_lce,
+        architecture=default_architecture,
+        operatingsystem=default_os,
+        content_view=module_promoted_cv,
+        group_parameters_attributes=[group_params],
+    ).create()
+    new_org = target_sat.api.Organization().create()
+    new_ak = target_sat.api.ActivationKey(organization=new_org).create()
+    with session:
+        session.hostgroup.update(
+            hg_name,
+            {
+                'activation_keys.activation_keys': module_ak_with_cv.name,
+            },
+        )
+        cmd = session.host.get_register_command(
+            {
+                'general.host_group': hg_name,
+                'advanced.rex_interface': iface,
+                'general.insecure': True,
+            },
+            full_read=True,
+        )
+        assert hg_name in cmd['general']['host_group']
+        assert module_ak_with_cv.name in cmd['general']['activation_key_helper']
+        assert constants.FAKE_0_CUSTOM_PACKAGE in cmd['advanced']['install_packages_helper']
+
+        session.organization.select(org_name=new_org.name)
+        cmd = session.host.get_register_command(
+            {
+                'general.organization': new_org.name,
+                'general.operating_system': default_os.title,
+                'general.insecure': True,
+            },
+            full_read=True,
+        )
+        assert new_org.name in cmd['general']['organization']
+        assert new_ak.name in cmd['general']['activation_keys']
+
+
+@pytest.mark.tier2
+@pytest.mark.usefixtures('enable_capsule_for_registration')
+@pytest.mark.no_containers
+def test_global_registration_with_gpg_repo_and_default_package(
+    session, module_activation_key, default_os, default_smart_proxy, rhel8_contenthost
+):
+    """Host registration form produces a correct registration command and host is
+    registered successfully with gpg repo enabled and have default package
+    installed.
+
+    :id: b5738b20-e281-4d0b-ac78-dcdc177b8c9f
+
+    :expectedresults: Host is successfully registered, gpg repo is enabled
+        and default package is installed.
+
+    :CaseLevel: Integration
+
+    :steps:
+        1. create and sync repository
+        2. create the content view and activation-key
+        3. update the 'host_packages' parameter in organization with package name e.g. vim
+        4. open the global registration form and update the gpg repo and key
+        5. check host is registered successfully with installed same package
+        6. check gpg repo is exist in registered host
+
+    :parametrized: yes
+    """
+    client = rhel8_contenthost
+    repo_name = 'foreman_register'
+    repo_url = settings.repos.gr_yum_repo.url
+    repo_gpg_url = settings.repos.gr_yum_repo.gpg_url
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.capsule': default_smart_proxy.name,
```

Comment:
```suggestion
```

Reply:
All the updates in test case will be addressed in  https://github.com/SatelliteQE/robottelo/pull/14001